app/src/test/java/io/apicurio/registry/ConfluentClientTest.java

```diff
@@ -13,6 +16,18 @@
 @QuarkusTest
 public class ConfluentClientTest {
 
+    private static H2DatabaseService h2ds = new H2DatabaseService();
```

Comment:
Same -- don't drag registry impl details into generic test.
DB is a choice of storage, not how CCompat should work!